app/src/components/menu.ts

```diff
@@ -286,6 +289,43 @@ export function createMenu({
     menuTemplate = [editMenu, viewMenu, windowMenu, helpMenu];
   }
 
+  try {
+    const bookmarkConfigPath = path.join(__dirname, '..', 'bookmarks.json');
+    if (fs.existsSync(bookmarkConfigPath)) {
+      const bookmarksConfig = JSON.parse(
```

Comment:
To replace later `['` access with friendlier `.` access and to avoid eventual whoopsies, please type your `bookmarksConfig as a `BookmarksConfig` expressed as the union of link or separator. Something like

```ts
type BookmarksLink = {
  type: 'link';
  title: string;
  url: string;
  shortcut?: string;
};
type BookmarksSeparator = {
  type: 'separator';
};
type BookmarksConfig = BookmarksLink | BookmarksSeparator;
```

Reply:
Done in 7c942cd179387cf000fad805095b2c154c318a03